### PR TITLE
New create topic flow

### PR DESCRIPTION
**Pull request by ana2201**
## Description
- Remove the big spinner currently shown when creating a new topic
- show the new topic in the list with a 'confirming' label
- redirect to new topic page once the tx is confirmed

## Screenshots
- preview link---> https://forum-staging-p47rdtk8v-usedispatch.vercel.app/

image

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -27,6 +27,12 @@
     ],
     "rules": {
         "no-console": "off",
+        "@typescript-eslint/array-type": [
+            "error",
+            {
+                "default": "array"
```
Comment (ana2201) [inline .eslintrc.json:33]: Changes this so that we could define arrays like `const x: string[] = ['a', 'b'];`

**Comment by zfaizal2** [comment]
I no longer see the revised "create forum" flow here, and I'm held up on this loading button until forum is created 
image

**Review by zfaizal2 — CHANGES_REQUESTED** [review]
mainly the comment above, I'll rereview it entirely once that's resolved

**Comment by ana2201** [comment]
> I no longer see the revised "create forum" flow here, and I'm held up on this loading button until forum is created image

Solved 🙂  the `useMemo` on forumContent was missing a dependency

**Review by diegorosolino — APPROVED** [review]

**Review by zfaizal2 — APPROVED** [review]

**Pull request merged**